class-two-factor-core.php

```diff
@@ -776,7 +776,7 @@ public static function filter_authenticate_block_cookies( $user ) {
 	 * @return boolean
 	 */
 	public static function is_user_api_login_enabled( $user_id ) {
-		return (bool) apply_filters( 'two_factor_user_api_login_enable', false, $user_id );
+		return (bool) apply_filters( 'two_factor_user_api_login_enable', (bool) did_action( 'application_password_did_authenticate' ), $user_id );
```

Comment:
This is a good action to use, I believe. It fires right after a _successful_ application passwords login.